salt/utils/timed_subprocess.py

```diff
@@ -4,6 +4,7 @@
 
 import subprocess
 import threading
+
```

Comment:
Not really. Just keeping consistent with pep standards :D happy to remove that commit